config/index.js

```diff
@@ -0,0 +1,5 @@
+const dev = process.env.NODE_ENV !== "production";
+
+export const server = dev
+  ? "http://localhost:3000"
+  : "https://playground-l9aacxvzj-yordaniss.vercel.app/";
```

Comment:
Im not sure if than will work cause this string "19aacxvzj" can be changes after next deployment

Reply:
I changed it to this: https://playground-jet.vercel.app. And got rid or '/' in the end